File: src/2455.average-value-of-even-numbers-that-are-divisible-by-three.2542/solution.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var averageValue = function (nums) {
    let sum = 0;
    let count = 0;
    for (const num of nums) {
        if (num % 6 === 0) {
            sum += num;
            count++;
        }
    }
    if (count === 0) {
        return 0;
    }
    return Math.floor(sum / count);
};

File: src/2455.average-value-of-even-numbers-that-are-divisible-by-three.2542/solution.java
class Solution {
    public int averageValue(int[] nums) {
        int sum = 0;
        int count = 0;
        for(int n : nums){
            if(n%6 == 0){
                sum += n;
                count++;
            }
        }
        return count == 0? 0:sum/count;
    }
}